app/AccountancyModule/Components/Participants/ParticipantList.php

```diff
@@ -191,64 +173,90 @@ function (Participant $p) use ($participantId) {
         $this->reload('Účastník byl odebrán', 'success');
     }
 
-    public function handleEditField(?int $aid = null, ?int $participantId = null, ?string $field = null, ?string $value = null) : void
+    public function handleEdit(int $participantId) : void
     {
-        if ($aid === null || $participantId === null || $field === null || $value === null) {
-            throw new BadRequestException();
+        if (! isset($this->participantsById()[$participantId])) {
+            throw new BadRequestException(
+                sprintf('Participant %d does not exist', $participantId),
+                IResponse::S404_NOT_FOUND
```

Comment:
404 mu nic asi neřekne, takže bud flashmessage nebo to logovat abychom vedeli ze k tomu někde dochází

Reply:
To by stálo za to logovat asi všude (explicitní vyhazování BadRequestException - protože to může znamenat problém v UI). Tohle totiž normálně nenastane, protože takový tlačítko by v UI být němelo.

Kdy to nastat může:
- uživatel ručně upraví URL - což je nám asi jedno, že failne
- uživatel v jiném tabu toho účastníka smaže

Mám radši dát redirect, kdy uživatel zůstane na stejné stránce a uvidí flash message, že účastník neexistuje?

Reply:
jo asi raději nyní redirect. Nekdy pak muzem hromadne zacit logovat stavy, co by neměli nastat